$MirOS: ports/security/ekeyd/patches/patch-daemon_ekey-rekey_8,v 1.2 2009/11/22 19:32:32 tg Exp $
--- host/ekey-rekey.8.orig	Wed Sep 14 14:58:09 2011
+++ host/ekey-rekey.8	Sun Jan 15 13:01:47 2012
@@ -18,7 +18,9 @@ Specifies the serial number of the key t
 Specifies the Master Key of the device being re-keyed.  If this is not specified, the user will be prompted for it.
 .TP
 .B DeviceNode
-If \-d or \-\-device is specified, then the tool expects to be give the device node to use (such as /dev/ttyACM0) rather than attempting to work it out from the serial number provided.
+If \-d or \-\-device is specified, then the tool expects to be given
+the device node to use (such as Linux /dev/ttyACM0 or BSD /dev/cuaU0)
+rather than attempting to work it out from the serial number provided.
 .SH "SEE ALSO"
 ekey-setkey(8), ekeyd(8)
 .SH AUTHOR
